experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -282,6 +283,8 @@ def answerSingleQuestion(self, question, imageId):
     def answerQuestionsFromFile(self, questionsFileName):
         questionFile = open(questionsFileName, 'r')
         for line in questionFile:
+            if self.header in line:
```

Comment:
I would propose skip not only header, but introduce some comment lines, for instance any line starting from ```#``` symbol is a comment:
```
# this is a comment
```